shiny/examples/todo_list/app.py

```diff
@@ -0,0 +1,119 @@
+import shinyswatch
+from htmltools import tags
+
+from shiny import App, module, reactive, render, ui
+
+app_ui = ui.page_fixed(
+    {"class": "my-5"},
+    tags.style(
+        """
+        .row {
+            border-bottom: 1px solid #eeeeee;
+            border-top: 1px solid #eeeeee;
+            margin-top: -1px;
+            padding: 20px 10px;
+            align-items: center;
+        }
+        .row p {
+        margin-bottom: 0px;
+        }
+        """
+    ),
+    shinyswatch.theme.minty(),
+    ui.panel_title("Shiny TodoMVC"),
+    ui.layout_sidebar(
+        ui.panel_sidebar(
+            ui.input_text("todo_input_text", "", placeholder="Todo text"),
+            ui.input_action_button("add", "Add to-do"),
+        ),
+        ui.panel_main(
+            ui.output_text("cleared_tasks"),
+            ui.div(id="tasks", style="margin-top: 0.5em"),
+        ),
+    ),
+)
+
+
+def server(input, output, session):
+    finished_tasks = reactive.Value(0)
+    task_counter = reactive.Value(0)
+
+    @output
+    @render.text
+    def cleared_tasks():
+        return f"Finished tasks: {finished_tasks()}"
+
+    @reactive.Effect
+    @reactive.event(input.add)
+    def add():
+        counter = task_counter.get() + 1
+        task_counter.set(counter)
+        id = "task_" + str(counter)
+        ui.insert_ui(
+            selector="#tasks",
+            where="afterBegin",
```

Comment:
This is the behavior I personally would expect from a todo app
```suggestion
            where="beforeEnd",
```